src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -134,5 +135,44 @@ public async Task RenewLicense()
                 context.Write(writer, renewLicense.ToJson());
             }
         }
+
+        [RavenAction("/license-server/connectivity", "GET", AuthorizationStatus.ValidUser)]
```

Comment:
should this also be a debug endpoint?

Reply:
We can add it to the debug package, but the timout needs to be small, 3-5 seconds max IMO.

Do we also want to have access to this during setup?

Reply:
done